deprecations.md

```diff
@@ -0,0 +1,48 @@
+# Deprecations
+
+## Upcoming
+
+### 31st March 2021
+
+#### Deprecated operations
+
+Following legacy operations will be completely removed:
+* `companies/getAllByName`
+* `bills/getAllByIds`
+* `bills/getAllByCustomers`
+* `bills/getAllClosed`
+* `creditCards/getAllByIds`
+* `creditCards/getAllByCustomers`
+* `reservations/getAllByIds`
+* `reservations/getAllByCustomers`
+* `reservations/getAllByNumbers`
+* `customers/getAllByIds`
+* `customers/getAllByEmails`
+* `customers/getAllByName`
+
+#### Deprecated time filters 
+
+Properties `TimeFilter`, `StartUtc` and `EndUtc` will be removed on operations:
+* [Get all customers](operations/customers.md#get-all-customers)
+* [Get all bills](operations/finance.md#get-all-bills)
+* [Get all companies](operations/enterprises.md#get-all-companies)
+* [Get all restrictions](operations/services.md#get-all-restrictions)
+* [Get all outlet items](operations/finance.md#get-all-outlet-items)
+* [Get all cashier transactions](operations/finance.md#get-all-cashier-transactions)
+
+Use properties `CreatedUtc`, `UpdatedUtc`, etc. as per operation description.
+
+#### Other deprecated properties
+
+Following properties will be removed from output and/or ignored on input:
+* `TaxIdentificationNumber` on [Company](operations/enterprises.md#company) 
+* `ChannelManagerId`, `TimeUnitCost`, `TraveAgencyId`, `ApplyCancelationFee` on [Reservation](operations/reservations.md#reservation)
```

Comment:
we deprecate apply cancellation fee ? + isnt is double L ? :D

Reply:
We are deprecating the one with typo 😄 the properly spelled one is there to stay. I'm editing this similarly to what Sylvia proposed for section above